inventories/nostr-rs-relay/inventory.yml

```diff
@@ -0,0 +1,22 @@
+---
+relay:
+  hosts:
+    olympics2024.nos.social:
+      relay_image_tag: olympics_latest
+    dev-olympics2024.nos.social:
```

Comment:
I think we can nix dev-olympics2024.nos.social, since we are trying to move to personal dev boxes rather than having one for every service. Is there an actual droplet here that we should tear down?

Reply:
tbh I feel weird about putting the olympics2024.nos.social relay in this inventory file too, because it wasn't deployed with the nostr-rs-relay playbook and tbh I'm not totally sure what would happen if we ran it on the live box. Maybe we can leave the vars in here but leave hosts empty with a comment that we don't have any production servers using this role right now?

Reply:
I'm ok with that. I didn't know we wanted personal servers but that's also fine. I had asked @gergelypolonkai to create a `dev-olympics2024.nos.social` box but we can just change the name to `daniel.dev.nos.social` instead. I assume the idea is reusing those for every type of test on other unrelated projects.

Reply:
I only added my personal host for the new inv and left olympics as before so the old inv and role works as it was. I have a relay deployed at my test ip but it still needs to have a dns name for the ssl to work, I think @gergelypolonkai is working on that.

Reply:
Oh yeah sorry I didn't communicate on the personal dev boxes. We were basically doubling our DO costs by having a dev version of each service and we really only used them to test ansible scripts.